### PR TITLE
[12.0][IMP] account: Payments without payment_id related aren't showed in...

### PR DESCRIPTION
**Pull request by carlosdauden**
…statement reconciliation

Same that https://github.com/OCA/OCB/pull/678, but v12.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

@Tecnativa TT20408

### CODE REVIEW

File: addons/account/models/reconciliation_widget.py [hunk 1]
```diff
@@ -477,12 +477,15 @@ def _domain_move_lines_for_reconciliation(self, st_line, aml_accounts, partner_i
             :param search_str:
         """
 
+        # Changes from odoo core to allow reconcile payment orders moves
```
Comment (pedrobaeza) [inline addons/account/models/reconciliation_widget.py]: This text is not needed

**Review by pedrobaeza — APPROVED** [review]
After talking directly with @carlosdauden and dig into the problem

**Comment by pedrobaeza** [comment]
@sbidoul we have here again the problem with blue lines presented in the reconciliation widget, but this time is deeper as supposedly Odoo introduced the threshold date for avoiding this, but it's worst:

- They are still restricting the blue lines to those with a linked payment.
- The date also filters black lines, so it prevents to reconcile imported AR/AP balances!

I don't think this is going to be attended by Odoo, so we decided to directly go for OCB removing the payment_id requirement + putting the threshold date only for blue lines.

**Review by sbidoul — APPROVED** [review]
I agree with the concept. Not reviewed deeply.

At some point we'll probably need to adapt account_payment_order to somehow generate account.payment objects.

**Comment by pedrobaeza** [comment]
OK, let's go with this for now: /ocabot merge

**Comment by pedrobaeza** [comment]
I didn't remember that we don't have Travis in this branch, so merging directly, and then cherry-picking to 13.0

**Comment by pedrobaeza** [comment]
Cherry-picked for v13 in https://github.com/OCA/OCB/commit/65b8da8b50c167e68945aafafd4c5e3f71ff8e44

**Pull request merged**